stl/inc/ostream

```diff
@@ -454,6 +454,13 @@ public:
         return *this;
     }
 
+#if _HAS_CXX20 // P1147R1 "Printing volatile Pointers"
+    template <class = void> // TRANSITION, ABI

```

Comment:
What exactly does `TRANSITION, ABI` mean here?

Reply:
Does it mean that it has to be a template member, since adding the usual member breaks ABI?

Reply:
Since this class is DLL-exported, its member functions are DLL-exported, and we can't add or remove exports without breaking ABI. The workaround for this is to add member function _templates_ instead of member functions.

Correction: We don't export this class. IIRC a prominent customer did so - despite the lack of support for doing so - and we broke them when adding the `nullptr_t` stream insertion operator. We gave them a grace period to fix the issue and are making new member functions templates in the interim until the next ABI break.